search-app/src/cmr/search/results_handlers/atom_json_results_handler.clj

```diff
@@ -78,6 +78,7 @@
                        :has_variables has-variables
                        :has_formats has-formats
                        :has_transforms has-transforms
+                       :has-spatial-subsetting has-spatial-subsetting
```

Comment:
We use underscores on our API so the keyword should have an underscore here.

Reply:
Whoops! Fixed.